styles.css

```diff
@@ -0,0 +1,99 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html {
+    font-family: 'Audiowide', cursive;
+}
+
+.main-wrapper {
+    height: 100vh;
+    width: 100vw;
+    background: #5433FF;  /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #A5FECB, #20BDFF, #5433FF);  /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #A5FECB, #20BDFF, #5433FF); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+    align-items: center;
+    justify-content: center;
+    display: flex;
+}
+
+.content-wrapper {
+    background-color: #ffffff;
+    height: 90%;
+    width: 70%;
+    border-radius: 4%;
+}
+
+header {
+    height: 15%;
+    width: 96%;
+    margin: auto;
+    text-align: center;
+    font-size: 35px;
+    padding: 25px;
+    background-image: linear-gradient(to top, #fc00ff, #00dbde); 
+    -webkit-background-clip: text;
+    color: transparent;
+    border-bottom: solid 3px #5433FF;
+}
+
+.round {
+    height: 7%;
+    text-align: center;
+    font-size: 35px;
+    padding: 15px;
+    color: #0066ff;
+}
+
+.game {
+    width: 96%;
+    height: 74%;
+    margin: auto;
+}
+
+.player {
+    float: left;
+    width: 49%;
+    padding: 15px;
+}
+
+.line {
+    float: left;
+    margin-top: 2%;
+    height: 91%;
+}
+
+#user-name {
```

Comment:
please, as I've showed you some time ago - don't use ids to styles. Only classes :)